src/PathFinder.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import {Button, Checkbox, form, FormControl, FormGroup, ControlLabel, ButtonToolbar } from 'react-bootstrap';
+
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import findPath from './PathFinderUtils'
+import _ from 'lodash'
+
+class PathFinder extends Component {
+
+  constructor(props) {
+    super()
+    if (localStorage.getItem('LedgerPathFinder')) {
+      this.state = JSON.parse(localStorage.getItem('LedgerPathFinder'))
+    } else {
+      this.state = {
+        done: false,
+        paused: false,
+        running: false,
+        batchSize: 5,
+        account: 0,
+        address: '',
+        result: [],
+        coin: 0,
+        index: 0,
+        segwit: false,
+      }
+    }
+    this.handleChangeAccount = this.handleChangeAccount.bind(this)
+    this.handleChangeAddress = this.handleChangeAddress.bind(this)
+    this.start = this.start.bind(this)
+    this.stop = this.stop.bind(this)
+    
+    this.reset = this.reset.bind(this)
+    this.handleChangeCoin = this.handleChangeCoin.bind(this)
+    this.handleChangeIndex = this.handleChangeIndex.bind(this)
+    this.handleChangeSegwit = this.handleChangeSegwit.bind(this)
+    this.onUpdate = this.onUpdate.bind(this)
+    this.onDone = this.onDone.bind(this)
+    this.onError = this.onError.bind(this)
```

Comment:
basically all the `.bind(this)` can be removed if you move the method to the `start = () => {}` syntax. 
see https://michalzalecki.com/react-components-and-class-properties/